### PR TITLE
EVENT-780-Improve-Sorts-Registration-View

### PR DESCRIPTION
**Pull request by TheNoodleMoose**

### CODE REVIEW

**Comment by wrandall22** [comment]
Putting our conversation onto "paper". This is basically ready from a front-end code standpoint, but waiting on the API side, right?

**Comment by TheNoodleMoose** [comment]
@wrandall22 Yes, this will have to wait til https://jira.cru.org/browse/EVENT-784 is completed.

File: app/scripts/controllers/eventRegistrations.js [hunk 1]
```diff
@@ -216,47 +216,25 @@ angular
...
-        } else if (orderBy === 'last_updated_timestamp') {
-          return registrant.lastUpdatedTimestamp;
-        } else if (orderBy === 'group_id') {
-          return registrant.groupId;
-        } else {
-          var answerValue = findAnswer(registrant, orderBy);
-          if (angular.isUndefined(answerValue)) {
-            return '';
-          }
+        var answerValue = findAnswer(registrant, orderBy);
```
Comment (TheNoodleMoose) [inline app/scripts/controllers/eventRegistrations.js:219]: I removed all of this as it was old filtering logic that the frontend was doing before API filtering was introduced. It was actually conflicting with the sorting that the API was doing.

File: app/views/eventRegistrations.html [hunk 2]
```diff
@@ -463,7 +479,7 @@ <h2 class="page-title" translate>Registrations</h2>
         </thead>
         <tbody>
           <tr
-            ng-repeat-start="r in registrants | orderBy:answerSort:reversesort"
+            ng-repeat-start="r in registrants | orderBy:answerSort"
```
Comment (TheNoodleMoose) [inline app/views/eventRegistrations.html:482]: I also removed ```reverseSort``` as it was doing an additional layer of reverse sorting on the frontend, even though the API is handling that now. It was essentially undoing the reverse sorting from the API.

**Review by wrandall22 — APPROVED** [review]

File: app/scripts/controllers/eventRegistrations.js [hunk 3]
```diff
@@ -499,36 +477,6 @@ angular
         return _.find($scope.registrations, { id: id });
       };
 
-      $scope.getRegistrationLastUpdatedTimestamp = (registrant) => {
```
Comment (TheNoodleMoose) [inline app/scripts/controllers/eventRegistrations.js:502]: We determined that we no longer need this function, as we are now just using the ```registrant.lastUpdatedTimestamp``` value instead.

**Comment by TheNoodleMoose** [comment]
@wrandall22 If this looks good then it can be deployed whenever @Benjamin-deVries merges his changes to production.

**Pull request merged**